Tribler/Test/Core/Upgrade/test_db72_to_pony.py

```diff
@@ -133,11 +132,9 @@ def test_cleanup_pony_experimental_db(self):
         mds.shutdown()
         shutil.copyfile(pony_db, pony_db_bak)
 
-        connection = sqlite3.connect(pony_db)
-        with connection:
+        with contextlib.closing(sqlite3.connect(pony_db)) as connection, connection:
```

Comment:
Ok, I admit. In two years of full-time Python programming, I've never seen context managers unfold to tuples. And I have never seen tuples unfolding like this (i.e. to the same variable). If one field of the tuple should be discarded, the convention is to assign it to `_`, like `as _, connection`.

Reply:
It's not a tuple, it's syntax sugar for nested "with" statements. Full answer is in the first conversation 😅